docs/documentation/algorithms/fidex/fidex.md

```diff
@@ -0,0 +1,503 @@
+# Fidex
+
+!!!warning
+    **This section is under construction and should not be considered as accurate yet.**
+
+## Description
+
+The `Fidex` algorithm is an approach to rule extraction that can be applied to neural network ensembles, decision Tree ensembles, and support vector machines. The name of the algorithm is a contraction of *fidelity* and *explainability*. *Fidelity* refers to how well an extracted rule mimics the behavior of a given model.
```

Comment:
Capitalization: "decision Tree". Shouldn't it be "Decision Tree" or "decision tree"?